src/azure-cli/azure/cli/command_modules/acs/tests/hybrid_2020_09_01/recordings/test_aks_create_blb_vmas.yaml

```diff
@@ -0,0 +1,116 @@
+      test@example.com\n"}]}}, "servicePrincipalProfile": {"clientId":"00000000-0000-0000-0000-000000000001",
+      "secret":"fake-secret"}, "addonProfiles": {}, "enableRBAC": true, "networkProfile":
+      {"networkPlugin": "kubenet", "podCidr": "10.244.0.0/16", "serviceCidr": "10.0.0.0/16",
+      "dnsServiceIP": "10.0.0.10", "dockerBridgeCidr": "172.17.0.1/16", "outboundType":
+      "loadBalancer", "loadBalancerSku": "basic"}}}'
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - aks create
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '1574'
+      Content-Type:
+      - application/json; charset=utf-8
+      ParameterSetName:
+      - --resource-group --name --location --dns-name-prefix --node-count --ssh-key-value
+        --service-principal --client-secret --load-balancer-sku --vm-set-type --no-wait
+      User-Agent:
+      - python/3.7.4 (Windows-10-10.0.19041-SP0) msrest/0.6.21 msrest_azure/0.6.3
+        azure-mgmt-containerservice/11.2.0 Azure-SDK-For-Python AZURECLI/2.23.0
+      accept-language:
+      - en-US
+    method: PUT
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/acscliapi2/providers/Microsoft.ContainerService/managedClusters/cliakstest000001?api-version=2020-11-01
+  response:
+    body:
+      string: "{\n  \"code\": \"QuotaExceeded\",\n  \"message\": \"Provisioning of
+        resource(s) for container service cliakstest000001 in resource group acscliapi2
+        failed. Message: Operation could not be completed as it results in exceeding
+        approved Total Regional Cores quota. Additional details - Deployment Model:
+        Resource Manager, Location: westus2, Current Limit: 20, Current Usage: 20,
+        Additional Required: 2, (Minimum) New Limit Required: 22. Submit a request
+        for Quota increase at https://aka.ms/ProdportalCRP/?#create/Microsoft.Support/Parameters/%7B%22subId%22:%228a4f688f-f361-4729-a197-1a8d663b603b%22,%22pesId%22:%2206bfd9d3-516b-d5c6-5802-169c800dec89%22,%22supportTopicId%22:%22e12e3d1d-7fa0-af33-c6d0-3c50df9658a3%22%7D
+        by specifying parameters listed in the \u2018Details\u2019 section for deployment
+        to succeed. Please read more about quota limits at https://docs.microsoft.com/en-us/azure/azure-supportability/regional-quota-requests..
+        Details: \"\n }"
```

Comment:
It looks like this test case is not executed correctly.

Reply:
Removed test not supported in Azurestack